.vscode/tasks.json

```diff
@@ -1,17 +1,74 @@
 {
-    "version": "0.1.0",
-    "command": "pwsh",
-    "isShellCommand": true,
-    "showOutput": "always",
-    "suppressTaskName": true,
-    "args": [ "-command" ],
+    "version": "2.0.0",
+
+    "windows": {
+        "options": {
+            "shell": {
+                "executable": "pwsh.exe",
+                "args": [
+                    "-NoProfile",
+                    "-ExecutionPolicy",
+                    "Bypass",
+                    "-Command"
+                ]
+            }
+        }
+    },
+    "linux": {
+        "options": {
+            "shell": {
+                "executable": "/usr/bin/pwsh",
+                "args": [
+                    "-NoProfile",
+                    "-Command"
+                ]
+            }
+        }
+    },
+    "osx": {
+        "options": {
+            "shell": {
+                "executable": "/usr/local/bin/pwsh",
+                "args": [
+                    "-NoProfile",
+                    "-Command"
+                ]
+            }
+        }
+    },
 
     "tasks": [
         {
-            "taskName": "build",
-            "args": [ "Import-Module ${workspaceRoot}/build.psm1; Start-PSBuild -Output ${workspaceRoot}/debug" ],
-            "isBuildCommand": true,
+            "label": "Bootstrap",
+            "type": "shell",
+            "command": "Import-Module ${workspaceFolder}/build.psm1; Start-PSBootstrap",
+            "problemMatcher": []
+        },
+        {
+            "label": "Clean Build",
+            "type": "shell",
+            "command": "Import-Module ${workspaceFolder}/build.psm1; Start-PSBuild -Clean",
+            "problemMatcher": []
```

Comment:
Doh! I forgot to put the $msCompile problem matcher here.  I could be wrong about this but Start-PSBuild -Clean doesn't **just** clean.  It cleans first and then builds, right?